prime-router/docs/design/design/topic.md

```diff
@@ -1 +1,61 @@
-TODO: Document ReportStream's Topic concept
\ No newline at end of file
+# Topic
+
+A topic consists of a name and a boolean indicating whether it's associated with the universal pipeline.  The primary
+intention is to silo data between different use cases.
+
+## Uses
+
+The primary use right now is to separate group of senders that route data through the universal pipeline and senders
+that route data through the legacy pipeline.  See below for more details.
+
+### Sender/Receivers
+
+- A sender has a configured topic and reports sent through the pipeline will be tagged with that topic
+- A receiver has configured topic and will only be routed reports that are tagged with that topic
+
+### Routing
+
+This is primary usage for topic and accomplishes the siloing, the routing step will only consider receivers that match
+the topic for the incoming messages.
+
+Additionally, each topic is configured with a different set of default filters that are applied if the receiver has not
+specifically defined their own.
+
+**TODO: remove the default filters associated with topics.  They are currently mostly unused within the universal pipeline and
```

Comment:
If there is agreement on the TODOs in this file, I will create separate follow up tickets for the cleanup

Reply:
like the idea of retire the unused feature, and receiver filters coming from more than one source could cause confusion and surprise.

yea, good to have prod involved since this is a feature already exposed to external users, in case there is user using it.